dnnv/nn/converters/tensorflow.py

```diff
@@ -227,20 +242,24 @@ def visit_Conv(self, operation):
         x_ = operation.x
         if isinstance(x_, Operation):
             x_ = self.visit(x_)
+        w_ = operation.w
+        if isinstance(w_, Operation):
+            w_ = self.visit(w_)
 
         @self._cached
         def conv_func(*inputs):
-            x = _concretize([x_], inputs)
+            x, weights = _concretize([x_, w_], inputs)
             if len(operation.kernel_shape) != 2:
                 raise NotImplementedError(
                     "Non 2d convolutions are not currently supported."
                 )
-            weights = operation.w
+            if not isinstance(weights, np.ndarray):
+                weights = np.array(weights)
```

Comment:
Is this necessary? What can it be other than a numpy array? Could it be `weights = np.asarray(weights)` instead of the if block?